assets/src/scss/layout/_page-header.scss

```diff
@@ -79,27 +79,40 @@
 }
 
 .page-header-background {
+  overflow: hidden;
   position: absolute;
   z-index: 0;
   top: 0;
   right: 0;
   left: 0;
 
+  _-- {
+    bottom: auto;
+  }
+
   &:after {
     content: "";
     display: block;
     position: absolute;
     z-index: 0;
-    height: 75%;
-    bottom: -.5%;
     left: 0;
     right: 0;
-    @include linear-gradient(360deg, $white 0%, transparent 100%);
+
+    _-- {
+      height: 75%;
+      bottom: -.5%;
+      background: linear-gradient(360deg, $white 0%, transparent 100%);
+    }
   }
 
   img {
+    _-- {
+      height: 100%;
+      @include mobile-only {
+        height: 100%;
```

Comment:
The Oceans theme needs to be able to set the height to `auto` on all sizes except mobile.